shell.nix

```diff
@@ -72,6 +72,7 @@ let
     yq
     z3
     zlib
+    lzma
```

Comment:
why do we need this?

Reply:
It's the requirement for `servant-swagger-ui`.

Reply:
`haskell.nix` should handle native dependencies automatically. What happens if you don't include this?

Reply:
Checked, it works without specifying it. Removed it.